bigtable/google/cloud/bigtable/table.py

```diff
@@ -27,7 +29,29 @@
 from google.cloud.bigtable.row import ConditionalRow
 from google.cloud.bigtable.row import DirectRow
 from google.cloud.bigtable.row_data import PartialRowsData
-
+from google.gax import RetryOptions, BackoffSettings
+from google.cloud.bigtable.retry import ReadRowsIterator, _create_row_request
+from grpc import StatusCode
+
+BACKOFF_SETTINGS = BackoffSettings(
```

Comment:
In particular we don't want a total timeout at all, really, since bigtable scans can take many minutes. I don't think the python library supports user-supplied deadlines but that's the only reason we'd want to stop a read.